backend/src/modules/player-profiles/core/application/command/CreatePlayerProfileCommandHandler.ts

```diff
@@ -25,6 +25,11 @@ export class CreatePlayerProfileCommandHandler implements CommandHandler<CreateP
       phoneNumber: command.phoneNumber,
     };
 
+    const allPlayers = await this.repository.findAll();
+    if (allPlayers.some((player) => player.emailAddress === command.emailAddress)) {
```

Comment:
Ogólnie to nie jest dobry pomysł. Bo między findAll, a createPlayerProfile mógłby się zarejestrowac ktoś z tym samym mailem. Przy dużym ruchu to nie zadziała. 

Ale jeśli nie umiecie inaczej to na teraz może być :) Ale normalnie to nigdy tak nie róbcie. 
Ale to jak ten test przechodzi skoro baza się nie czyści? Tutaj jest to samo praktycznie.

Reply:
Test przechodzi, bo w WriteSide omijamy wywołanie commandHandlera, tylko idzie od razu zapis do bazy... a normalnie w aplikacji z frontu przejdzie przez to przechodzi, Si ?

Reply:
Nie rozumieć co Twoja mówić do moja :P